docs/02-knora-ontologies/knora-base.md

```diff
@@ -7,605 +7,459 @@
 
 ## Overview
 
-The Knora base ontology is the main built-in Knora ontology. Each
-project that uses DSP-API must describe its data model by creating
-ontologies that extend this ontology.
+The Knora base ontology is the main built-in Knora ontology. Each project that uses DSP-API must describe its data model
+by creating ontologies that extend this ontology.
 
-The Knora base ontology is identified by the IRI
-`http://www.knora.org/ontology/knora-base`. In the DSP-API documentation
-in general, it is identified by the prefix `knora-base`, but for
-brevity, in this document, we use `kb` or omit the prefix entirely.
+The Knora base ontology is identified by the IRI `http://www.knora.org/ontology/knora-base`. In the DSP-API
+documentation in general, it is identified by the prefix `knora-base`, but for brevity, in this document, we use `kb` or
+omit the prefix entirely.
 
 ## The Knora Data Model
 
-The Knora data model is based on the observation that, in the
-humanities, a value or literal is often itself structured and can be
-highly complex. Moreover, a value may have its own metadata, such as its
-creation date, information about permissions, and so on. Therefore, the
-Knora base ontology describes structured value types that can store this
-type of metadata. In the diagram below, a book (`ex:book2`) has a title
+The Knora data model is based on the observation that, in the humanities, a value or literal is often itself structured
+and can be highly complex. Moreover, a value may have its own metadata, such as its creation date, information about
+permissions, and so on. Therefore, the Knora base ontology describes structured value types that can store this type of
+metadata. In the diagram below, a book (`ex:book2`) has a title
 (identified by the predicate `ex:title`) and a publication date
 (`ex:pubdate`), each of which has some metadata.
 
 ![Figure 1](knora-base-fig1.dot.png "Figure 1")
 
 ### Projects
 
-In Knora, each item of data belongs to some particular project. Each
-project using Knora must define a `kb:knoraProject`, which has these
-properties (cardinalities are indicated in parentheses after each
-property name):
+In Knora, each item of data belongs to some particular project. Each project using Knora must define a `kb:knoraProject`
+, which has these properties (cardinalities are indicated in parentheses after each property name):
```

Comment:
Move `,` to previous line.

Reply:
yes, Intellij's markdown auto formatting sucks at times... I fixed most cases but not all